### DISCUSSION TITLE
Delete subscriptions from control panel

### DISCUSSION THREAD

**Discussion by lukeholder** [craftcms/commerce#2343] [Ideas]
Should be able to manually delete a subscription which will unsubscribe the user first, then delete.

Add delete buttons to subscriptions on customer info tab. Also check they have permissions to do so.

**Comment by lukeholder** [comment]
Deleting a subscription will not be possible due to subscriptions not being deletable in Stripe.

**Comment by sanderpotjer** [comment]
But you can have subscriptions not related to Stripe 😄
